config/routes.rb

```diff
@@ -7,5 +7,12 @@
   patch '/teachers/:id' => 'teachers#update'
   delete '/teachers/:id'=> 'teachers#destroy'
 
+  get '/'             => 'students#index'
```

Comment:
A routes page can only have one root url, so having two ```'/'``` will cause a conflict because Rails won't know which controller action to use. Make sure that as part of a team, you guys make these kinds of decisions together. Don't just talk to your pair programming partner, talk to everyone on the team to ensure you're all on the same page.